PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,20 @@
 ## Change Summary
 
-> Description here
+> Link to Jira ticket if applicable
+> Describe the changes (even if there is a description in Jira)
+> Identify backwards incompatible changes; link to tickets/PRs that must merge or deploy first
+> Call out known issues, high risk areas, or future work to be done
+> Include guidance for reviewers if relevant
```

Comment:
Some suggestions for other useful data points:

- Describe any new dependencies that this pull requests adds to the project, such as new libraries, new Github or Secret Manager secrets, or environment variables
- If this pull request is dependent on new work being done on a different project, please point to the relevant branch/PR/release.